crates/toolchain/build/src/lib.rs

```diff
@@ -19,8 +19,11 @@ pub use self::config::GuestOptions;
 
 mod config;
 
-#[allow(dead_code)]
 const RUSTUP_TOOLCHAIN_NAME: &str = "nightly-2024-10-30";
+const OPENVM_BUILD_LOCKED_ENV: &str = "OPENVM_BUILD_LOCKED";
```

Comment:
I think the OPENVM_ prefix is redundant in the constant names (since this is in our codebase already) -- a nice perk of adding these constants